### PR TITLE
CLDR-16201 Add initial unitPreferenceData for rainfall-rate, snowfall-rate

### PR DESCRIPTION
**Pull request by pedberg-icu**
CLDR-16201

- [ ] This PR completes the ticket.

Add initial unitPreferenceData for rainfall-rate, snowfall-rate. Can refine if we get more input on this.

### CODE REVIEW

**Review by markusicu — APPROVED** [review]
looks plausible to me

with the usual caveat that when we write regions="... US ..." we should probably include most of the US territories/possessions (PR, VI, ...) but that's for another day

**Review by richgillam — APPROVED** [review]
Looks good to me...

File: common/supplemental/units.xml [hunk 1]
```diff
@@ -443,6 +443,15 @@ For terms of use, see http://www.unicode.org/copyright.html
             <unitPreference regions="001">kilometer-per-hour</unitPreference>
             <unitPreference regions="GB US">mile-per-hour</unitPreference>
         </unitPreferences>
+        <unitPreferences category="speed" usage="rainfall-rate">
```
Comment (sffc) [inline common/supplemental/units.xml:446]: Does it need to be "rainfall-rate", or could it just be "rainfall"?

**Comment by macchiati** [comment]
I agree. The usage parameters are already specific to the quantity (length,
area, etc) so we don't need additional qualifiers.

**Pull request merged**